js/api.js

```diff
@@ -0,0 +1,5 @@
+export const fetchData = async (dataCategory) => {
+    const data = await fetch(`http://localhost:3000/${dataCategory}`);
+    const returnData = await data.json();
+    return returnData;
```

Comment:
The variable names could use a bit of work -- 

```
export const fetchCategory = (categoryId) => {
  const response = await fetch(...);
  const body = await response.json();

  return body;
}
```

Why not inlining everything altogether:

```
export const fetchCategory = (categoryId) => 
  fetch('...').then(response => response.json())
```